docs/preview/02-Features/02-message-handling/04-event-hubs-azure-functions.md

```diff
@@ -0,0 +1,241 @@
+﻿---
+title: "Azure EventHubs message handling for Azure Functions"
+layout: default
+---
+
+This article describes how you can use Arcus' message handler concept with Azure Functions; allowing you to more easily port your business logic from/to Azure Functions.
+
+# Azure EventHubs message handling for Azure Functions
+Our EventHubs message pump system provides a way to receive, route, and handle incoming events, but this is not needed in an Azure Functions environment.
```

Comment:
Only receiving is not needed in an Azure Functions environment ?

Reply:
Changed to only receiving 👍